.travis.yml

```diff
@@ -15,6 +15,9 @@ cache: bundler
 script:
   - bundle update
   - bundle exec rake pedant
+env:
+  global:
+    - "PEDANT_OPTS=\"--skip-oc_id\""
```

Comment:
this should just land in https://github.com/chef/chef-zero/blob/master/spec/run_oc_pedant.rb#L149 since it'll always be needed.